src/Client/GuzzleFileFetcher.php

```diff
@@ -84,8 +84,10 @@ public function fetchMetaData(string $fileName, int $maxBytes): PromiseInterface
     /**
      * {@inheritDoc}
      */
-    public function fetchTarget(string $fileName, int $maxBytes, array $options = []): PromiseInterface
+    public function fetchTarget(string $fileName, int $maxBytes, array $options = [], string $url = null): PromiseInterface
```

Comment:
Even though we are using `@inheritdoc` can we still document the extra `@param`'s somehow?

Reply:
Sure! Documentation added.